### PR TITLE
Initialize Monkey Game

### LINKED ISSUE #45: Monkey Game Setup and Styling
**Is your feature request related to a problem? Please describe.**
Making a game with monkeys in it.

### PR DESCRIPTION
**Pull request by krferrell**
## Changes
1. Import Images for the game
2. Create components to render the images in the game
3. Created a game loop to move the images

## Purpose
This is the start of the game's skeleton. 

## Approach
Used the React-Native-Game-Engine (RNGE) to render and move the images around the screen 

Closes #45

### CODE REVIEW

File: components/kennyGame/KGameEngine.js [hunk 1]
```diff
@@ -31,22 +44,70 @@ const KGamePage = () => {
...
+      monkey3.position[0] += 2;
+      if (monkey3.position[0] === -50) {
+        setMonkeyBool3(false);
+      }
+    } else if (!monkeyBool3) {
+      monkey3.position[0] -= 2;
+      if (monkey3.position[0] === -450) {
+        setMonkeyBool3(true);
+      }
+    }
```
Comment (Lap343) [inline components/kennyGame/KGameEngine.js:87]: Monkey

File: components/kennyGame/FeedMonkey.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    </View>
+  );
+};
+
+const styles = StyleSheet.create({
+    feedMonkey: {
+      height: 200,
+      width: 0,
+      borderRadius: 50,
+    //   backgroundColor: "red",
```
Comment (Lap343) [inline components/kennyGame/FeedMonkey.js]: comment still needed?

File: components/kennyGame/GrabberMonkey.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  );
+};
+
+const styles = (props) =>
+  StyleSheet.create({
+    bird: {
+      height: 200,
+      width: 100,
+      borderRadius: 50,
+      // backgroundColor: "red",
```
Comment (connielion) [inline components/kennyGame/GrabberMonkey.js]: still need this?

File: components/kennyGame/KGameEngine.js (hunk 1 above)
Comment (JorgeLVilla) [inline components/kennyGame/KGameEngine.js:87]: Monkeys eating the logic.

**Review by JorgeLVilla — APPROVED** [review]
Great job Kenny!

**Pull request merged**